### DISCUSSION TITLE
看不懂啊，特别是中文的新闻。

### DISCUSSION THREAD

**Discussion by Aphirotty** [AI4Finance-Foundation/FinGPT#14] [Q&A]
> $ py shares_news_sentiment_classify.py

====
读下方新闻，列举3个可能受影响的美国股票，分别简短地判断'积极'或'消极':

====
{
  "choices": [
    {
      "finish_reason": "length",
      "index": 0,
      "logprobs": null,
      "text": "\n\u65b0\u95fb\uff1a\u7f8e\u56fd\u603b\u7edf\u7279\u6717\u666e\u5ba3\u5e03\uff0c\u7f8e\u56fd\u5c06\u5bf9\u4e2d\u56fd\u8fdb\u53e3\u5546\u54c1\u5f81\u6536\u5173\u7a0e\u3002"
    }
  ],
  "created": 1686985507,
  "id": "cmpl-7SKHbor10BBEZIIswTtR9HAppgXRp",
  "model": "text-davinci-003",
  "object": "text_completion",
  "usage": {
    "completion_tokens": 64,
    "prompt_tokens": 80,
    "total_tokens": 144
  }
}

====
Read following news and list 3 stocks that may be affected, briefly judge each one 'positive' or 'negative':

This Company Has the World’s Most Downloaded Shopping App. So Why Aren’t U.S. Retailers Worried?,
Meghan Markle's podcast isn't coming back for a second season, and she and Harry aren't getting the full $20 million payout from,
NAB shares offer franked dividends, but is NAB cheap?,

====
{
  "choices": [
    {
      "finish_reason": "length",
      "index": 0,
      "logprobs": null,
      "text": "\n1. Walmart: Positive - Walmart is the world's most downloaded shopping app, so this news could be seen as a positive for the company.\n\n2. Spotify: Negative - Meghan Markle's podcast is not coming back for a second season, which could be seen as a negative for Spotify.\n"
    }
  ],
  "created": 1686985509,
  "id": "cmpl-7SKHd9ytkagLY0Mt7YF9kcHNAPXXp",
  "model": "text-davinci-003",
  "object": "text_completion",
  "usage": {
    "completion_tokens": 64,
    "prompt_tokens": 104,
    "total_tokens": 168
  }
}

看不懂什么意思，特别是中文的新闻，作为一个技术小白，太难为我了

**Comment by YangletLiu** [comment]
How about starting by playing with our sample/demo scripts?   Like the examples on OpenAI's API webpage: https://platform.openai.com/examples

**Reply by Aphirotty** [reply]
Thanks, I'll try。 Peace and health on Dragon Boat Festival

**Reply by YangletLiu** [reply]
3Q, U2!

**Reply by lihongteng** [reply]
我也是想利用到量化交易软件上去，让他用自动去交易期货，但是目前有也是看不懂。训练的话我侧重成交数据推理就可以了，不参考消息面因素。

**Reply by YangletLiu** [reply]
从例程出发，比较能行。
